backend/models/course.model.ts

```diff
@@ -25,6 +25,18 @@ export interface CourseVisibilityAttributes {
   published?: boolean;
 }
 
+const ModuleSchema: Schema = new Schema({
+  title: String,
```

Comment:
I'm wondering if we should make the title required for the modules?

Reply:
Good call, I'll switch it